delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResolversRoutes.scala

```diff
@@ -167,23 +176,30 @@ final class ResolversRoutes(
       }
     }
 
-  private def resolve(resourceSegment: IdSegmentRef, projectRef: ProjectRef, resolverId: Option[IdSegment])(implicit
+  private def resolve(
+      resourceSegment: IdSegmentRef,
+      projectRef: ProjectRef,
+      resolverId: Option[IdSegment],
+      asSource: Boolean = false
+  )(implicit
       caller: Caller
   ): Route =
     authorizeFor(projectRef, Permissions.resources.read).apply {
       parameter("showReport".as[Boolean].withDefault(default = false)) { showReport =>
-        def emitResult[R: JsonLdEncoder](io: IO[MultiResolutionResult[R]]) =
+        def emitResult[R: JsonLdEncoder](io: IO[MultiResolutionResult[R]]) = {
           if (showReport)
             emit(io.map(_.report).attemptNarrow[ResolverRejection])
+          else if (asSource)
+            emit(io.map(_.value.source).attemptNarrow[ResolverRejection])
           else
             emit(io.map(_.value.jsonLdValue).attemptNarrow[ResolverRejection])
+        }
```

Comment:
There is a question here about combinations of `showReport` and `asSource` but I think this works and personally I don't think the refactor needed is worth it